tests/e2e/tests/e2e/OpenshiftConnector.spec.ts

```diff
@@ -66,12 +66,15 @@ suite('Openshift connector user story', async () => {
         }
       ];
 
+    await driverHelper.navigateToUrl(TestConstants.TS_SELENIUM_BASE_URL);
+    await loginPage.login();
     await testWorkspaceUtils.createWsFromDevFile(wsConfig);
   });
 
   test('Login into workspace and open plugin', async () => {
+    await driverHelper.reloadPage();
```

Comment:
It would be nice to add the comment to issue which `reloadPage()` action is fixing.